src/types/CloudProviders.ts

```diff
@@ -0,0 +1,35 @@
+import { BridgeResponse } from "@rhoas/smart-events-management-sdk";
+
+type RegionDetails = {
+  value: string;
+  label: string;
+};
+export type CloudProviderDetails = {
+  value: string;
+  label: string;
+  region: RegionDetails[];
+};
+
+const cloudProviderOptions: CloudProviderDetails[] = [
+  {
+    value: "aws",
+    label: "Amazon Web Services",
+    region: [{ value: "us-east-1", label: "US East, N. Virginia" }],
+  },
+];
+
+export const getUpdatedCloudProviders = (
```

Comment:
Probably in this context, this function name makes not much sense. WDYT about `getCloudProviderAndRegionForInstance` or `getCloudProviderAndRegionByInstance`?

Reply:
Yes, Okay
Thanks @vpellegrino